core/lib/makeswift/components/customer-group-slot/customer-group-slot.makeswift.tsx

```diff
@@ -0,0 +1,123 @@
+import { Checkbox, Combobox, Group, List, Slot, Style } from '@makeswift/runtime/controls';
+import { ReactNode } from 'react';
+import useSWR from 'swr';
+
+import { CustomerGroupType, CustomersGroup } from '~/app/api/customer/groups/route';
+import { GetCustomerGroupResponse } from '~/client/queries/get-customer';
+import { runtime } from '~/lib/makeswift/runtime';
+
+async function fetchData(): Promise<CustomerGroupType> {
```

Comment:
I think there was a previous comment about making this function name more explicit and agree. `getCustomerGroupType()`?